File: NRZ.Web/client/src/models/eServicesSettings.ts
export class EServiceSettingsModel {
    constructor(obj: any = {}) {
        this.serviceProviderName = obj.serviceProviderName;
        this.serviceProviderBank = obj.serviceProviderBank;
        this.serviceProviderBic = obj.serviceProviderBic;
        this.serviceProviderIban = obj.serviceProviderIban;
        this.currency = obj.currency;
        this.seizedPropertyCertificateRequestExpirationDays = obj.seizedPropertyCertificateRequestExpirationDays || undefined;
        this.seizedPropertyCertificateFee = obj.seizedPropertyCertificateFee || undefined;
        this.seizedPropertyCertificateReason = obj.seizedPropertyCertificateReason;
        this.seizedPropertyReportRequestExpirationDays = obj.seizedPropertyReportRequestExpirationDays || undefined;
        this.seizedPropertyByOwnerReportFee = obj.seizedPropertyByOwnerReportFee || undefined;
        this.seizedPropertyReportReason = obj.seizedPropertyReportReason;
        this.administrativeServiceNotificationUrl = obj.administrativeServiceNotificationUrl;
    }

    serviceProviderName: string;
    serviceProviderBank: string;
    serviceProviderBic: string;
    serviceProviderIban: string;
    currency: string;
    seizedPropertyCertificateRequestExpirationDays?: number;
    seizedPropertyCertificateFee?: number;
    seizedPropertyCertificateReason: string;
    seizedPropertyReportRequestExpirationDays?: number;
    seizedPropertyByOwnerReportFee?: number;
    seizedPropertyReportReason: string;
    administrativeServiceNotificationUrl: string;
}

File: NRZ.Data/EservicesSettings.cs
using System;
using System.Collections.Generic;

namespace NRZ.Data
{
    public partial class EservicesSettings
    {
        public string ServiceProviderName { get; set; }
        public string ServiceProviderBank { get; set; }
        public string ServiceProviderBic { get; set; }
        public string ServiceProviderIban { get; set; }
        public string Currency { get; set; }
        public int? SeizedPropertyCertificateRequestExpirationDays { get; set; }
        public decimal? SeizedPropertyCertificateFee { get; set; }
        public string SeizedPropertyCertificateReason { get; set; }
        public int? SeizedPropertyReportRequestExpirationDays { get; set; }
        public decimal? SeizedPropertyByOwnerReportFee { get; set; }
        public string SeizedPropertyReportReason { get; set; }
        public string AdministrativeServiceNotificationUrl { get; set; }
    }
}
